oak_functions_launcher/src/lookup.rs

```diff
@@ -33,9 +33,18 @@ pub async fn update_lookup_data(
     let max_chunk_size = ByteUnit::Gibibyte(2);
 
     let lookup_data = load_lookup_data(lookup_data_path)?;
-    let chunk = encode_lookup_data(lookup_data, max_chunk_size)?;
+    let mut chunks = chunk_up_lookup_data(lookup_data, max_chunk_size);
+
+    if chunks.len() != 1 {
+        return Err(anyhow!(
+            "lookup data size exceeds: {:?} bytes",
+            max_chunk_size
+        ));
+    }
```

Comment:
I thought we were trying to get multiple chunks (the issue even says `n` chunks). Or is this PR not addressing #3719?

Reply:
Clarified in a comment, but this was meant to be a small PR just to compute the chunks, but not to send them yet.